src/AbstractActionHandler.ts

```diff
@@ -28,7 +28,10 @@ export abstract class AbstractActionHandler {
     const { blockInfo } = block
 
     if (isRollback || (isReplay && isFirstBlock)) {
-      await this.rollbackTo(blockInfo.blockNumber - 1)
+      const rollbackCount = this.lastProcessedBlockNumber - blockInfo.blockNumber - 1
```

Comment:
Flip these two lines, and you get:
```
const rollbackBlockNumber = blockInfo.blockNumber - 1
const rollbackCount = this.lastProcessedBlockNumber - rollbackBlockNumber
```
which seems a little bit more descriptive of what's happening, plus it pushes the actual value that you're using into a bit more prominence.